From 46ccd0fab64ff045143e12f1af5c1ea966ec17b7 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Sun, 12 Jun 2016 09:37:47 -0700
Subject: [PATCH] Add .got.plt to testsuite/script_test_2.t

The .got.plt section must be placed right after the .got section.
Otherwise, GOT offset will be wrong.

	PR gold/20246
	* testsuite/script_test_2.t: Add .got.plt after .got.
---
 gold/testsuite/script_test_2.t | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gold/testsuite/script_test_2.t b/gold/testsuite/script_test_2.t
index 0baa428..0ce9408 100644
--- a/gold/testsuite/script_test_2.t
+++ b/gold/testsuite/script_test_2.t
@@ -33,6 +33,7 @@ SECTIONS
   . = ALIGN(0x100);
   .data : { *(.data) }
   .got : { *(.got .toc) }
+  .got.plt : { *(.got.plt) }
   .bss : { *(.bss) }
 
   /* Now the real test.  */
-- 
2.5.5

